packages/serve-runtime/src/useCompleteSubscriptionsOnDispose.ts

```diff
@@ -16,16 +17,7 @@ export function useCompleteSubscriptionsOnUnifiedGraphDispose(
                   // eslint-disable-next-line @typescript-eslint/no-floating-promises
                   stop.then(() => result.return?.());
                   onDispose(() => {
-                    stop(
-                      createGraphQLError(
-                        'subscription has been closed because the server is shutting down',
-                        {
-                          extensions: {
-                            code: 'SHUTTING_DOWN',
-                          },
-                        },
-                      ),
-                    );
+                    stop(createError());
```

Comment:
No need to define the error outside.
```suggestion
                    stop(
                      createGraphQLError(
                        'subscription has been closed because the server is shutting down',
                        {
                          extensions: {
                            code: 'SHUTTING_DOWN',
                          },
                        },
                      ),
                    );
```